packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -195,21 +188,33 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
     }
   }
 
-  private _onRenderPreview = (previewProps: IEditorPreviewProps, defaultRender: (props: IEditorPreviewProps) => React.ReactNode) => {
-    const content = defaultRender({ ...previewProps, isScrollable: true });
-    if (this._activeCustomizations) {
-      return (
-        <CustomizerContext.Provider value={{ customizations: { settings: {}, scopedSettings: {} } }}>
-          <Customizer {...this._activeCustomizations}>
-            <ThemeProvider scheme={_schemes[this.state.schemeIndex]}>
-              <Stack styles={regionStyles}>{content}</Stack>
-            </ThemeProvider>
-          </Customizer>
-        </CustomizerContext.Provider>
-      );
+  // tslint:disable-next-line:member-ordering
+  private _getPreviewComponent = memoizeFunction(
```

Comment:
For memoizing React Functional Components, I think we should be switching to React.memo these days

Reply:
I'm not sure React.memo would work in this case?